CODEOWNERS

```diff
@@ -3,73 +3,73 @@
 
 # Only files explicitly mentioned are guarded by code owners (no * rule).
 
-*           @digital-asset/daml
+*           @digital-asset/empty
```

Comment:
@cbaatz We should leave an instruction to never remove the `empty` group from anything. Then people can add themselves to it, if they want to be notified about everything.

Reply:
but then rather rename it to `all-notifications` or something in that spirit.